src/Api/END_POINTS.tsx

```diff
@@ -0,0 +1,11 @@
+const BASE_URL = "https://upskilling-egypt.com:3000/api/v0";
```

Comment:
We can create an axios interceptor and include in it the base url and both the authorization header and the accept language.
This will eliminate the need to include these headers in every axios calls.

Reply:
done